dd-sdk-android-core/build.gradle.kts

```diff
@@ -109,6 +109,7 @@ dependencies {
     // Generate NoOp implementations
     ksp(project(":tools:noopfactory"))
 
+    implementation(project(":dd-sdk-android-internal"))
```

Comment:
there is already such dependency declared few lines above

Reply:
removed